content-src/components/ContextMenu/ContextMenu.scss

```diff
@@ -33,8 +34,33 @@
         &:hover {
           background: $star-blue;
           color: $white;
+
+          a {
+            color: $dark-grey;
+          }
+
+          a:hover {
+            color: $white;
+          }
         }
       }
     }
   }
 }
+
+.submenu {
+  position: relative;
+
+  .context-menu {
+    display: none;
+    left: auto;
+    right: 95%;
```

Comment:
You have to do it by reading the position of the element relative to the window :S definitely a follow-up...